src/components/_global/BalPopover/BalPopover.vue

```diff
@@ -114,8 +116,10 @@ watch(popoverOpened, () => {
 }
 
 .align-center-transform {
-  -webkit-transform: translateX(-50%);
-  transform: translateX(-50%);
+  -webkit-transform: translateX(
+    -webkit-calc(-50% + v-bind(activatorHalfWidthPx))
+  );
+  transform: translateX(calc(-50% + v-bind(activatorHalfWidthPx)));
```

Comment:
The centered position looks better when its shifted by half of the width of the activator.